src/Accelerators/NNPA/Pass/DLCPasses.hpp

```diff
@@ -20,9 +20,6 @@
 namespace mlir {
```

Comment:
I think we should avoid placing our not official dialects in the `mlir` namespace. Suggest to use `onnx_mlir` namespace at minimum, and better to place each dialect conversion passes in its own namespace nested inside onnx_mlir ? 

See the krnl dialect conversions for a template.

Reply:
Agreed. Will have a separate PR for this. Anyway, this part is not code in this patch.